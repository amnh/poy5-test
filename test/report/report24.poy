(* Build and report terminals *)
set(seed:50, timer:0)
read ("1.fas", "30.ss" )
build (1)
report (terminals)
