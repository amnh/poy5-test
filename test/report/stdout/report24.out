Terminals Files:
  
Input File: 1.fas
  Terminals Included (49)
  
    t9
    t8
    t7
    t6
    t5
    t49
    t48
    t47
    t46
    t45
    t44
    t43
    t42
    t41
    t40
    t4
    t39
    t38
    t37
    t36
    t35
    t34
    t33
    t32
    t31
    t30
    t3
    t29
    t28
    t27
    t26
    t25
    t24
    t23
    t22
    t21
    t20
    t2
    t19
    t18
    t17
    t16
    t15
    t14
    t13
    t12
    t11
    t10
    t1
    
  Terminals Excluded (0)
  
    
  

Input File: 30.ss
  Terminals Included (49)
  
    t9
    t8
    t7
    t6
    t5
    t49
    t48
    t47
    t46
    t45
    t44
    t43
    t42
    t41
    t40
    t4
    t39
    t38
    t37
    t36
    t35
    t34
    t33
    t32
    t31
    t30
    t3
    t29
    t28
    t27
    t26
    t25
    t24
    t23
    t22
    t21
    t20
    t2
    t19
    t18
    t17
    t16
    t15
    t14
    t13
    t12
    t11
    t10
    t1
    
  Terminals Excluded (0)
  
    
  
